packages/shield-controller/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export type { CoverageStatus } from './types';
+export type {
+  ShieldControllerActions,
+  ShieldControllerEvents,
+  AllowedActions as ShieldControllerAllowedActions,
+  AllowedEvents as ShieldControllerAllowedEvents,
+  ShieldControllerMessenger,
+  ShieldControllerState,
+} from './ShieldController';
+export { ShieldController } from './ShieldController';
```

Comment:
It looks like the controller defines one action and one event, can you export those here so that other clients and packages can make use of them?

Also can you export the "get default state" function so that it can be used in other places as well?

``` suggestion
  ShieldControllerState,
  ShieldControllerCheckCoverageAction,
  ShieldControllerCoverageResultReceivedEvent
} from './ShieldController';
export {
  ShieldController,
  getDefaultShieldControllerState,
} from './ShieldController';
```